package.json

```diff
@@ -18,28 +18,34 @@
     "utils-merge": "1.0.1"
   },
   "devDependencies": {
+    "@types/node": "^10.12.18",
     "after": "0.8.2",
     "eslint": "3.19.0",
     "eslint-plugin-markdown": "1.0.0-beta.6",
     "finalhandler": "1.1.1",
     "istanbul": "0.4.5",
     "mocha": "2.5.3",
-    "supertest": "1.1.0"
+    "supertest": "1.1.0",
+    "typescript": "^2.0.0"
   },
   "files": [
     "lib/",
     "LICENSE",
     "HISTORY.md",
     "README.md",
-    "index.js"
+    "index.js",
+    "index.d.ts"
   ],
+  "main": "index.js",
```

Comment:
Please remove the `main` prop; it's not needed.

Reply:
done